### DISCUSSION TITLE
Adding Cameras to Apple Homekit

### DISCUSSION THREAD

**Discussion by tekwreck** [tsightler/ring-mqtt#395] [Support Q&A]
I am able to add the ring cameras via Home Assistant to my Apple Home. The snapshots refresh the still image about every 10 seconds which is great but I can't seem to reliably get the Live Feed to come up via Apple Home. I'm getting INVALIDDATA errors when I look at my HA Logs after trying to start a live view.

```
Logger: homeassistant.components.stream.stream.camera.driveway_camera
Source: components/stream/__init__.py:347
Integration: Stream (documentation, issues)
First occurred: 12:15:34 AM (78 occurrences)
Last logged: 8:42:38 AM

Error from stream worker: Error opening stream (INVALIDDATA, Invalid data found when processing input) rtsp://03cabcc9-ring-mqtt:8554/{camera_id}_live
```

It seems to bug out as well even when I am trying to load the camera as well on my HA Dashboard on my browser too after trying to play a stream via Apple Home.

Has anyone had any experience with using this addon to add cameras to Apple Home?

**Comment by tsightler** [comment]
While the issue you are posting does not seem to have anything specifically to do with Homekit, if you are wanting to use Ring cameras with Homekit I would highly recommend the use of [Homebridge Ring](https://github.com/dgreif/ring/tree/master/homebridge) or [Scrypted](https://github.com/koush/scrypted) vs trying to use this project in conjunction with HA  and Homekit as those other projects have native support for Homekit.
